src/components/pageHeader/index.tsx

```diff
@@ -28,27 +28,37 @@ const StyledSubtitle = styled(Paragraph)`
   line-height: 32px;
   margin-top: -40px;
   color: ${(props: StyledSubtitleProps) =>
-    props.textColor ? props.textColor : { DARK_GREY }};
+    props.subtitleColor ? props.subtitleColor : { DARK_GREY }};
 `;
 
+interface PageHeaderProps extends StyledSubtitleProps {
+  readonly pageTitle: string;
+  readonly isMobile?: boolean;
+  readonly pageSubtitle?: string;
+}
+
 const PageHeader: React.FC<PageHeaderProps> = ({
   pageTitle,
+  isMobile,
   pageSubtitle,
   subtitleColor,
 }) => {
-  const { windowType } = useWindowDimensions();
-  return (
-    <div>
-      <StyledTitle
-        style={{
-          marginTop: `${windowType === WindowTypes.Mobile ? '0vh' : '1vh'}`,
-        }}
-      >
-        {pageTitle}
-      </StyledTitle>
-      <StyledSubtitle textColor={subtitleColor}>{pageSubtitle}</StyledSubtitle>
-    </div>
-  );
+  if (isMobile) {
+    return (
+      <div>
```

Comment:
⛏️ These `<div>` elements can be react fragments, which might make the final DOM tree look a little nicer.